storybook/src/components/card/card.core-template.ts

```diff
@@ -22,7 +22,14 @@ export const coreCard: ComponentImplementation<Card<never>> = {
           interactions.length > 0 &&
           html`<div slot="interactions">
             ${interactions.map(
-              (interaction) => html`<div class="dso-card-interaction">${buttonTemplate(interaction)}</div>`
+              (interaction) => html`
+                <div class="dso-card-interaction">
+                  ${interaction.type === "button" ? buttonTemplate(interaction) : nothing}
+                  ${interaction.type === "toggletip"
+                    ? html`${toggletipTemplate(interaction)} ${interaction.label}`
```

Comment:
`${interaction.label}` zou ik weghalen, dat is geen onderdeel van de toggleTip die we demo'en.